<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTx</class>
 <widget class="QMainWindow" name="FTx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTx</string>
  </property>
  <widget class="QWidget" name="cw">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="spl">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="vl_1">
        <item>
         <layout class="QHBoxLayout" name="hl_2">
          <item>
           <widget class="QLineEdit" name="ledFind">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFind">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_3">
          <item>
           <widget class="QLineEdit" name="ledDir"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnDir">
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tblList"/>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="txtShow"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
